CONTRIBUTING.md

```diff
@@ -109,12 +109,68 @@ Follow GitHub's fork-branch-pull request pattern.
 
 4. For any changes to existing API's, alter the unit tests to cover the changes (and remove tests made irrelevant due to your changes).
 
-5. Review the static code analyses results. 
+5. Review the static code analyses results from the Static Analysis and CodeQL Analysis workflows. For more information on how to use these workflows, view [Using GitHub Actions Workflows](#using-github-actions-workflows).  
     - Push code changes to the appropriate forked repository. 
     - Go to the Actions tab and enable GitHub Actions Workflow. The CodeQL Analyis and Static Analysis will be triggered for all pushed code. 
     - Review these workflows for any warnings or errors. 
     - Once successful, create a pull request. 
 
+### Using GitHub Actions Workflows
```

Comment:
Maybe add something about how these can be seen in their forks?

Reply:
Fixed. 
> All of our workflows will be available for forked repositories once enabled. To enable workflows, navigate to the Actions tab and click "I understand my workflows, go ahead and enable them".